lab_1/main.py

```diff
@@ -51,7 +76,11 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    if not isinstance((language, text), str) or not isinstance(stop_words, list):
```

Comment:
` isinstance((language, text), str)` does not work like that. `isinstance` supports only one object at a time. Split into several checks.